default/stringtables/en.txt

```diff
@@ -4866,8 +4866,10 @@ At %planet%: depopulation is complete.
 EFFECT_CONC_CAMP_COMLETE_LABEL
 Ethnic Cleansing
 
-EFFECT_DRYDOCK_SHIP_REPAIR
-On %planet%: the %building% repaired the %ship%.
+EFFECT_DRYDOCK_SHIP_REPAIR_PARTIAL
+On %planet%: partial repairs were performed on the %ship% by the %building%.
+EFFECT_DRYDOCK_SHIP_REPAIR_COMPLETE
+On %planet%: the %building% completed repairs on the %ship%.
 EFFECT_DRYDOCK_SHIP_REPAIR_LABEL
 Drydock Repair
 
```

Comment:
The repair label needs updating for both, as far as I know it needs separate labels else you get two identical options to select/unselect.